webapp/src/sui.tsx

```diff
@@ -59,7 +60,7 @@ export function fireClickOnEnter(e: React.KeyboardEvent): void {
     let charCode = (typeof e.which == "number") ? e.which : e.keyCode
     if (charCode === 13 || charCode === 32) {
         e.preventDefault();
-        (document.activeElement as HTMLElement).click();
+        (e.currentTarget as HTMLElement).click();
     }
```

Comment:
This was a part of the changes I had to do to support the focus in the dropdown menu items. The fact is that previously, I used "document.activeElement" because I didn't founded the "target" or "currentTarget" of React.KeyboardEvent, probably because I didn't searched enough.
Now, in the behavior of an Item of a DropDownMenu, when we hit Enter, we give the focus back to the DropDownMenu, when we close the modal, the focus will be given to the menu again. But the problem is that this function here is executed on a keyDown on each item, it will look for "document.activeElement" and do a click on it... In our case, the activeElement is now the DropDownMenu, not the item. That's why I do this change.
I checked the behavior of the rest of the website, and I didn't found any regression with this change.